cluster/local/integration_tests.sh

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
Do you think we should refactor this, and move the common part to build repository? It feels like only line 260 onward should be in this repo

Reply:
I was wondering the same.  It may be easier to bring this benefit to the other crossplaneio managed stacks if it were in the build project.

As a standalone test, this may be easier to inform stack developers on how to create integration tests of their own, however the stack-azure build process is already dependent on the crossplaneio/build project.  Because of the dependency, I don't think this potential value stands.  I agree that this would be best homed in crossplaneio/build.

I would be fine with a merge into this repo if we wanted to follow up with the relocation later.

Reply:
It seems like line 207 onward may be necessary to keep independent. However, I am not sure any of it is generic enough to be in the build submodule. It might be useful to get this implemented and then investigate how we can standardize on it across stacks, I definitely have some ideas. Update the build submodule can result in long cycles as I know you are aware of :)